src/tensors/levels/sparselistlevels.jl

```diff
@@ -150,7 +150,11 @@ end
 summary_fiber_abbrev(lvl::VirtualSparseListLevel) = "sl($(summary_fiber_abbrev(lvl.lvl)))"
 
 function virtual_level_size(lvl::VirtualSparseListLevel, ctx)
-    ext = Extent(literal(lvl.Ti(1)), lvl.shape)
+    if lvl.Ti <: Integer 
```

Comment:
Could we add a TODO comment here saying that `we need to convert this to a multiple dispatch approach`?